lib/snowflexid/protocol.ex

```diff
@@ -1,5 +1,7 @@
-defmodule SnowflexId.IdHelper do
-  @moduledoc false
+defmodule SnowflexId.Protocol do
```

Comment:
The concept of Protocol wouldn't be a good fit here because it's meant for different implementations while in this case the algorithm that generates ID is only one.

Reply:
I`m not sure about what name I could use for this module. Feel free to commit whichever name you prefer on this branch

Reply:
Id again seems fine.